core/src/main/resources/config.yml

```diff
@@ -115,6 +115,10 @@ economy:
   # Used to replace %currency% for the language files and the 2nd hologram line.
   server-currency: $
   use-xp: false
+  taxes:
+    sell-to-shop-tax-percent: 0
+    buy-from-shop-tax-percent: 0
+    collecting-player-name: ""
```

Comment:
tax officer name would better? Not sure if we have to even include this since it's just taxes and everyone knows the server is collecting the tax.